app/src/androidTest/java/org/mozilla/fenix/ui/robots/ThreeDotMenuMainRobot.kt

```diff
@@ -357,6 +370,14 @@ private fun threeDotMenuRecyclerViewExists() {
 private fun settingsButton() = mDevice.findObject(UiSelector().text("Settings"))
 private fun assertSettingsButton() = assertTrue(settingsButton().waitForExists(waitingTime))
 
+private fun customizeHomeButton() =
+    mDevice.findObject(
+        UiSelector()
+            .resourceId("$packageName:id/text")
+            .textContains("Customize home")
```

Comment:
and here ^

Reply:
Please see my comment.

Reply:
`    <string name="browser_menu_customize_home">Customize home</string>`